### PR TITLE
Fixed help string in CRUD documentation.

### LINKED ISSUE #4134: archives named borg-test-data
the help text in the code talks about "archives named borg-test-data", but the de-facto archives names are different.

master branch fix needed + maybe backports to 1.1 and 1.0 if there is same issue.

### PR DESCRIPTION
**Pull request by Ashmodei**
Fixes #4134 
Help text in code has been fixed and appropriate documentation files been rebuilt.

### CODE REVIEW

File: src/borg/archiver.py [hunk 1]
```diff
@@ -515,6 +515,7 @@ def create_inner(archive, cache, fso):
         dry_run = args.dry_run
         t0 = datetime.utcnow()
         t0_monotonic = time.monotonic()
+        logger.info('Creating archive at "%s"' % args.location.orig)
```
Comment (ThomasWaldmann) [inline src/borg/archiver.py]: this change does not belong into this PR.

File: docs/man/borg-benchmark-crud.1 [hunk 2]
```diff
@@ -1,6 +1,6 @@
 .\" Man page generated from reStructuredText.
 .
-.TH BORG-BENCHMARK-CRUD 1 "2017-11-25" "" "borg backup tool"
+.TH BORG-BENCHMARK-CRUD 1 "2018-11-12" "" "borg backup tool"
```
Comment (ThomasWaldmann) [inline docs/man/borg-benchmark-crud.1]: no need to update/commit the generated files, this is done at release time.

**Comment by Ashmodei** [comment]
OK, I deleted wrong commits.

**Comment by ThomasWaldmann** [comment]
Thanks!

**Pull request merged**